include/mbgl/map/map.hpp

```diff
@@ -127,6 +129,15 @@ class Map : private util::noncopyable {
     void setAccessToken(const std::string &token);
     const std::string &getAccessToken() const;
 
+    // Projection
+    inline void getWorldBoundsMeters(ProjectedMeters &sw, ProjectedMeters &ne) const { Projection::getWorldBoundsMeters(sw, ne); }
```

Comment:
Instead of passing out args via reference, there should be a `template <class T> class Bounds` and the return type of this function should be `Bounds<ProjectedMeters>`.